README.md

```diff
@@ -197,25 +197,62 @@ This plugin creates device plugin endpoints based on the configurations given in
-| "isRdma"         | N        | Mount RDMA resources                   | `bool` - boolean value true or false                                                               | "isRdma": true                                                                                                         |
 
 Note: "resourceName" must be unique only in the scope of a given prefix, including the one specified globally in the CLI params, e.g. "example.com/10G", "acme.com/10G" and "acme.com/40G" are perfectly valid names.
 
-[//]: # (The table above generated using: https://ozh.github.io/ascii-tables/)
+#### Device selectors
+Each selector evaluated in order as listed below in selector tables below.
+
+#### Common selectors
+All device types support following common device selectors.
+
+|   Field   | Required |                Description                |         Type/Defaults          |   Example/Accepted values   |
+|-----------|----------|-------------------------------------------|--------------------------------|-----------------------------|
+| "vendors" | N        | Target device's vendor Hex code as string | `string` list Default: `null`  | "vendors": ["8086"]         |
+| "devices" | N        | Target Devices' device Hex code as string | `string` list Default: `null`  | "devices": ["154c", "10ed"] |
+| "drivers" | N        | Target device driver names as string      | `string` list Default: `null`  | "drivers": ["vfio-pci"]     |
+
+
+#### Extended NetDevice selectors
+In addition to the common selectors from above table, "netDevice" also supports following selectors.
+
+|     Field     | Required |                          Description                           |                   Type/Defaults                   |                               Example/Accepted values                                |
+|---------------|----------|----------------------------------------------------------------|---------------------------------------------------|--------------------------------------------------------------------------------------|
+| "pfNames"     | N        | VFs from PF matches list of PF names                           | `string` list Default: `null`                     | "pfNames": ["enp2s2f0"] (See follow-up sections for some advance usage of "pfNames") |
+| "linkTypes"   | N        | The link type of the net device associated with the PCI device | `string` list Default: `null`                     | "linkTypes": ["ether"]                                                               |
+| "ddpProfiles" | N        | A map of device selectors                                      | `string` list Default: `null`                     | "ddpProfiles": ["GTPv1-C/U IPv4/IPv6 payload"]                                       |
+| "isRdma"      | N        | Mount RDMA resources                                           | `bool`  values `true` or `false` Default: `false` | "isRdma": `true`                                                                     |
```

Comment:
It might be good to also update example yaml files under /docs and /deployments that are still using `isRdma` as field of `ResourceConfig`.

```
./docs/rdma/configMap.yaml
./docs/dpdk/mlnx-dpdk/configMap.yaml
./docs/dpdk/mlnx-dpdk/configMap.yaml
./deployments/configMap.yaml
```

Reply:
I will update them too. thanks